schema/playground/launcher.schema

```diff
@@ -20,19 +21,31 @@ type LaunchMeCommandRequest {
     improbable.Vector3f impact_point = 1;
     improbable.Vector3f launch_direction = 2;
     float launch_energy = 3;
+    int64 launcher = 4;
 }
 
 type LaunchMeCommandResponse {}
 
+type ScoreIncreaseRequest {
+    int64 launcher_to_score = 1;
+    float amount_increase = 2;
+}
+
+type ScoreIncreaseResponse {}
+
 component Launcher {
     id = 12005;
     float energy_left = 1;
     float recharge_time_left = 3;
+    float score = 4;
```

Comment:
Should probably refactor this to its own `Score` component

Reply:
+1

Reply:
Also in a different file. A component is meant to represent some set of behaviour on that entity. Score and launch are quite different although one consumes the other.

Reply:
launcher is about shooting something, right? (I find the name very confusing)
 this component is mainly about shooting and `energy_left` directly defines whether you are able to shoot or not.

The score is a result of shooting, but is not as tightly coupled as the shooting energy. So I agree that this should be refactored into a `Score` component

Reply:
I agree about the naming being confusing.

And sure enough, I'll refactor it, then.